glide.lock

```diff
@@ -385,6 +385,8 @@ imports:
   - spdy
 - name: github.com/elazarl/go-bindata-assetfs
   version: 3dcc96556217539f50599357fb481ac0dc7439b9
+- name: github.com/elazarl/goproxy
+  version: c4fc26588b6ef8af07a191fcb6476387bdd46711
```

Comment:
@juanvallejo figure out what added this.  I don't see other corresponding changes.  Can you just reset your entire branch to release-3.11 and start over?

Reply:
Per off-thread conversation, this package was moved from the `testImports` section when `./hack/update-deps.sh` was run.